fe/fe-core/src/main/java/com/starrocks/load/loadv2/LoadJob.java

```diff
@@ -818,16 +818,22 @@ public TLoadInfo toThrift() {
             // progress
             switch (state) {
                 case PENDING:
-                    info.setProgress("ETL:0%; LOAD:0%");
+                    info.setProgress("ETL:0%; LOAD:0%; PUBLISH:0%");
```

Comment:
since it has only 2 states, how about
```suggestion
                    info.setProgress("ETL:0%; LOAD:0%; PUBLISHED: [TRUE|FALSE|N/A]");
```